src/containers/PatientDetails/PatientDocument/PatientDocumentFooter/index.tsx

```diff
@@ -0,0 +1,59 @@
+import { t } from '@lingui/macro';
+import { QuestionnaireResponse } from 'fhir/r4b';
+import { useCallback, useContext } from 'react';
+import { useFormContext } from 'react-hook-form';
+import { calcInitialContext } from 'sdc-qrf';
+
+import { RemoteDataResult } from '@beda.software/remote-data';
+
+import { BaseQuestionnaireResponseFormPropsContext } from 'src/components/BaseQuestionnaireResponseForm/context';
+import { FormFooter, FormFooterComponentProps } from 'src/components/BaseQuestionnaireResponseForm/FormFooter';
+
+import { S } from './styles';
+
+export interface PatientDocumentFooterProps extends FormFooterComponentProps {
+    saveButtonTitle?: React.ReactNode;
+    saveDraftButtonTitle?: React.ReactNode;
+    onSaveDraft?: (questionnaireResponse: QuestionnaireResponse) => Promise<RemoteDataResult<QuestionnaireResponse>>;
+}
+
+export function PatientDocumentFooter(props: PatientDocumentFooterProps) {
```

Comment:
@alexlipovka  There is no need for extra PatientDocumentFooter component.
Please move all logic to the existing FormFooter component.